scala/scala_import.bzl

```diff
@@ -53,14 +57,14 @@ def _create_provider(
         neverlink,
         source_jar,
         intellij_metadata):
-    transitive_runtime_jars = [
+    new_transitive_runtime_jars = [
```

Comment:
You are only changing the name of this where you create it not where it is used. That seems like a bug to me.

Reply:
It was just part of trying to understand the variables used here. This method is not used anymore so I removed it.